### PR TITLE
Add sv and sv-SE locales to support translations list

### PR DESCRIPTION
**Pull request by bjankord**
### Summary
Part of the work needed for [adding translation support for sv and sv-SE](https://github.com/cerner/terra-core/issues/1700).

### CODE REVIEW

**Review by tbiethman — APPROVED** [review]

**Review by StephenEsser — APPROVED** [review]

**Comment by emilyrohrbough** [comment]
Do we need to wait to add these locales until the repositories consuming them contain these translations?

**Review by emilyrohrbough — APPROVED** [review]

**Review by nfaltermeier — APPROVED** [review]

**Review by ryanthemanuel — APPROVED** [review]

**Comment by bjankord** [comment]
Waiting until translation repos land in other repos before merging this down.

**Review by ryanthemanuel — APPROVED** [review]

**Pull request merged**
